README.md

```diff
@@ -232,8 +232,8 @@ is enough, it's easy as doing this:
 auto loop = uvw::Loop::getDefault();
 ```
 
-Note that loop objects don't require to be closed explicitly, even if they offer
-the `close` member function in case an user wants to do that.<br/>
+Note that loop objects don't require being closed explicitly, even if they offer
+the `close` member function in case a user wants to do that.<br/>
```

Comment:
Shouldn't it be _an user_ rather than _a user_?

Reply:
the a/an difference is usually in the sound, rather than the letter. So, for example, we'd probably use "an umbrella", cause it starts with a vowel sound, whereas something like "a university", cause it starts with more of a consonant sound ("yooniversity"). Here, user sounds like "yooser", so that's why it would normally go with "a".

Reply:
Got it. Thank you for the explanation. :+1: